packages/block-library/src/stretchy-text/index.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * Stretchy Text block registration.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Registers the `core/stretchy-text` block on the server.
+ *
+ * @since 6.9.0
+ *
+ * @uses render_block_core_stretchy_text()
+ * @throws WP_Error An WP_Error exception parsing the block definition.
+ **/
+function register_block_core_stretchy_text() {
+	// TODO - only enqueue the script if the block is used.
+	wp_enqueue_script_module( '@wordpress/block-library/stretchy-text/view' );
```

Comment:
Is this the best way to add a module? Should we start supporting the `viewFile` key for core blocks?